connector/profiler/src/main/scala/org/apache/spark/executor/profiler/package.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.executor
+
+import java.util.concurrent.TimeUnit
+
+import org.apache.spark.internal.config.ConfigBuilder
+
+package object profiler {
+
+  private[profiler] val EXECUTOR_PROFILING_ENABLED =
+    ConfigBuilder("spark.executor.profiling.enabled")
+      .doc("Turn on code profiling via async_profiler in executors.")
+      .version("4.0.0")
+      .booleanConf
+      .createWithDefault(false)
+
+  private[profiler] val EXECUTOR_PROFILING_DFS_DIR =
+    ConfigBuilder("spark.executor.profiling.dfsDir")
```

Comment:
@dongjoon-hyun, I'm also investigating adding profiling support for the driver, should I fork all configurations, or simply treat the driver as a special executor and resue those configurations?

Reply:
Well, @pan3793 , could you use a new JIRA for your new goal, please?

Although I understand why you adds a comment here, it doesn't look like a good practice to me. Why do we add a comment about driver-related suggestion to the irrelevant executor-related PR like `Spark Executor JVM Profiling`?